CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## v1.1.0 (March 2021)
 
 ### Updates
- - [#56](https://github.com/aws/device-shadow-for-aws-iot-embedded-sdk/pull/56), [#56](https://github.com/aws/device-shadow-for-aws-iot-embedded-sdk/pull/72) and [#73](https://github.com/aws/device-shadow-for-aws-iot-embedded-sdk/pull/73) Add support for Named Shadow feature of AWS IoT Device Shadow service.
+ - [#56](https://github.com/aws/device-shadow-for-aws-iot-embedded-sdk/pull/56), [#56](https://github.com/aws/device-shadow-for-aws-iot-embedded-sdk/pull/72) and [#73](https://github.com/aws/device-shadow-for-aws-iot-embedded-sdk/pull/73) Add support for Named Shadow, a feature of AWS IoT Device Defender service that allows you to create multiple shadows for a single IoT device.
```

Comment:
Add support for named shadow, a feature of the AWS IoT Device Defender service that allows you to create multiple shadows for a single IoT device.

Reply:
Updated